keps/sig-storage/1885-csi-migration-azurefile/kep.yaml

```diff
@@ -33,6 +33,7 @@ latest-milestone: "v1.24"
 milestone:
   alpha: "v1.15"
   beta: "v1.21"
+  stable: "v1.26"
```

Comment:
Please also update line 30 from `latest-milestone: "v1.24"` to `latest-milestone: "v1.26"`